communication/src/main/java/org/jboss/da/communication/aprox/impl/AproxConnectorImpl.java

```diff
@@ -83,6 +85,7 @@ public AproxConnectorImpl(Configuration configuration) {
         }
 
         try {
+            userLog.info("Retrieving versions for maven artifacts " + ga + " from " + query);
             log.info("Retrieving maven metadata for " + ga + " from " + query);
```

Comment:
No need to duplicate the logs, user logs end up with others, only the logger name is different.